devices/mini.go

```diff
@@ -0,0 +1,73 @@
+package devices
+
+import (
+	"image"
+
+	streamdeck "github.com/magicmonkey/go-streamdeck"
+)
+
+var (
+	miniName                     string
+	miniButtonWidth              uint
+	miniButtonHeight             uint
+	miniImageReportPayloadLength uint
+	miniImageReportHeaderLength  uint
+	miniImageReportLength        uint
+)
+
+// GetImageHeaderMini returns the USB comms header for a button image for the XL
+func GetImageHeaderMini(bytesRemaining uint, btnIndex uint, pageNumber uint) []byte {
```

Comment:
Reworked the entire function somewhat. This is based off code in the python API client StreamDeck Mini device file. The header was pretty simple to figure out once I started capturing USB packets with Wireshark.